iptables/table_test.go

```diff
@@ -739,6 +794,7 @@ func describeDirtyDataplaneTests(appendMode bool, dataplaneMode string) {
 				InsertMode:               insertMode,
 				BackendMode:              dataplaneMode,
 				LookPathOverride:         lookPathNoLegacy,
+				TopLevelChainsOverride:   set.From("INPUT", "OUTPUT", "FORWARD", "PREROUTING", "POSTROUTING", "unexpected-insert"),
```

Comment:
I don't think it's right to override the top-level chains here; I think the test found the bug; the approach I gave you can't inserts to unexpected chains :-(